pkg/cloudprovider/providers/azure/azure_storage.go

```diff
@@ -19,23 +19,49 @@ package azure
 import (
 	"fmt"
 
+	"github.com/Azure/azure-sdk-for-go/arm/storage"
+	"github.com/Azure/go-autorest/autorest/to"
 	"github.com/golang/glog"
 )
 
+const (
+	defaultStorageAccountType  = string(storage.StandardLRS)
+	fileShareAccountNamePrefix = "f"
```

Comment:
better use a full naming like 'fileshare'. 'f' is also a hex digital and may get mixed with uuids.

Reply:
`f` would be better, it saves 3 letters.